internal/provider/resource_project.go

```diff
@@ -78,10 +75,9 @@ API: https://api-docs.neon.tech/reference/createproject`,
 			"history_retention_seconds": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				Default:      providerDefaultHistoryRetentionSeconds,
 				ValidateFunc: intValidationNotNegative,
 				Description: `The number of seconds to retain the point-in-time restore (PITR) backup history for this project.
-Default: 7 days, see https://neon.tech/docs/reference/glossary#point-in-time-restore.`,
+The default on Neon when creating a project is 1 day. See also https://neon.tech/docs/reference/glossary#point-in-time-restore.`,
```

Comment:
**(issue)** Please remove this note from the docstring, the link to the official documentation is enough to avoid discrepancies in future, i.e., repeating the issue you are solving with this change 😀

Reply:
Done!